tox.ini

```diff
@@ -47,6 +47,7 @@ commands =
   py{27,34,35,36,37}-unit: py.test --quiet --cov={envdir}/opencensus --cov=context --cov=contrib --cov-report= --cov-config=.coveragerc --cov-fail-under=97 tests/unit/ context/ contrib/
   ; TODO: System tests
   py37-lint: flake8 context/ contrib/ opencensus/ tests/ examples/
+  py37-lint: bash ./scripts/pylint.sh
```

Comment:
This assumes all developers are using Linux/Mac?

Reply:
Please refer to #361, where metrics didn't work on Windows.
Having the CI running on both Linux and Windows could help to catch such issues.

Reply:
I don't expect developers to run the lint environment directly from their machines. 
Travis default environment is Ubuntu Linux so the scripts should run under builds as long as we don't change the vm. 
https://docs.travis-ci.com/user/reference/overview/

Reply:
Would you update https://github.com/census-instrumentation/opencensus-python/blob/master/CONTRIBUTING.md#instructions in this PR to show how local test would work?

Reply:
@reyang what do you recommend for cross-platform scripts? I agree that our tox targets shouldn't assume the platform is linux.

Reply:
I would write a small Python script in this case.

Reply:
Yes that's a good idea. Will make a separate PR to address this issue.